kolibri/core/assets/src/exams/utils.js

```diff
@@ -82,6 +82,32 @@ function annotateQuestionsWithItem(questions) {
   });
 }
 
+/* Given a V2 question_sources, return V3 structure with those questions within one new section */
+export function convertV2toV3(questionSources = []) {
+  return {
+    section_title: '', // TODO Get a default string for the section title in here that is translated
+    description: '',
+    resource_pool: [],
+    questions: annotateQuestionsWithItem(questionSources),
```

Comment:
Per my comment below, would also need to return a randomization boolean here, based on the quiz's boolean value for randomization.